workspaces/web/src/components/NavBar.svelte

```diff
@@ -26,7 +26,14 @@
   <div class="navbar-end">
     <div class="navbar-item">
       <div class="buttons">
-        <GitHubButton />
+		<Button href="http://github.com/sponsors/kantord" size="small" outlined inverted info>
```

Comment:
Instead of having this button right here, please move it to a new file, similarly how GitHubButton is implemented

Reply:
Should be ok right now